{"name": "testcase2", "status": "failed", "statusDetails": {"message": "AssertionError: assert 2 == 3\n  +2\n  -3", "trace": "@pytest.mark.run(order=2)\n    @allure.title(\"testcase2\")\n    @allure.description(\"I am first test_case2\")\n    @allure.story('story_2')\n    def test_case2():\n        print('I am case2')\n        LoggerForApitest.writeInfoLog(\"I AM CASE2\")\n>       assert 2 == 3\nE       assert 2 == 3\nE         +2\nE         -3\n\npytest_case\\test_fixtures1.py:48: AssertionError"}, "description": "I am first test_case2", "attachments": [{"name": "log", "source": "267683de-cc00-45ea-87da-7b673d8fc143-attachment.txt", "type": "text/plain"}, {"name": "stdout", "source": "7a3af842-6a14-4165-9b76-5c2e1776b0e7-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "974359f0-d038-4538-b735-c8fa3d49af1b-attachment.txt", "type": "text/plain"}], "start": 1647999876051, "stop": 1647999876052, "uuid": "0ed3df0e-9c7e-4f89-8d69-149d075a0cc2", "historyId": "92f9d3b642298a60c88279ecf235fa2d", "testCaseId": "725ce7cbcf31b4144284ab868efa6e15", "fullName": "pytest_case.test_fixtures1#test_case2", "labels": [{"name": "story", "value": "story_2"}, {"name": "tag", "value": "run(order=2)"}, {"name": "parentSuite", "value": "pytest_case"}, {"name": "suite", "value": "test_fixtures1"}, {"name": "host", "value": "no-delecs-g42-test-wx405299"}, {"name": "thread", "value": "15464-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "pytest_case.test_fixtures1"}]}